src/components/Posts/PostsMaker/PostMakerContainer/PostMakerContainer.tsx

```diff
@@ -90,18 +93,21 @@ export default function PostMakerContainer({
       </div>
 
       {/* Content */}
-      <div>{children}</div>
+      <div className="my-4">{children}</div>
       {/* Category */}
-      <div className="p-px border-t">
-        <div className="flex justify-start gap-3 items-center px-3">
-          <p className="text-color-primary text-base font-medium"># Topic</p>
+      <div className=" border-t border-gray-200 dark:border-gray-700">
```

Comment:
remove the whitespace as the begging " border-t..."

Reply:
yep